tracer/src/Datadog.Tracer.Native/il_rewriter.cpp

```diff
@@ -776,7 +776,7 @@ bool ILRewriter::IsLoadLocalDirectInstruction(unsigned opcode)
     }
 }
 
-int ILRewriter::GetLocalIndexFromOpcode(const ILInstr* pInstr)
+unsigned ILRewriter::GetLocalIndexFromOpcode(const ILInstr* pInstr)
```

Comment:
prefer using `unsigned int` or better `std::uint32_t` (https://en.cppreference.com/w/cpp/types/integer) to make it clear.

Reply:
Fixed in 6956445deeebc1d843cfd9568ac599a1339df503